index.html

```diff
@@ -2297,14 +2302,15 @@
           <li>Return <var>P</var>, but continue running these steps <a>in
           parallel</a>.
           </li>
-          <li>Let <var>list</var> be a new <a>PresentationConnectionList</a>.
-          </li>
-          <li>Resolve <var>P</var> with <var>list</var>.
+          <li>If the <a>set of presentation controllers</a> contains at least
+          one <a>presentation connection</a>, <a>resolve</a> <var>P</var> with
+          the <a>PresentationConnectionList</a> object associated with the <a>
+            PresentationReceiver</a> object.
           </li>
-          <li>If the <a>user agent</a> is not <a>monitoring incoming
-          presentation connections</a>, start <a>monitoring incoming
-          presentation connections</a> from <a>controlling browsing
-          contexts</a>.
+          <li>Otherwise, <var>P</var> remains unsettled. It will be resolved
```

Comment:
OK, I used suggested wording, associating the promise with the `PresentationReceiver` instead of the `PresentationConnectionList` as that seemed more logic.

The step is not a real step in any case, it really is there to alert readers that it's normal if the promise remains unsettled at the end of that algorithm.